src/pageEditor/analysisManager.ts

```diff
@@ -51,8 +51,7 @@ const runtimeActions = runtimeSlice.actions;
 const pageEditorAnalysisManager = new ReduxAnalysisManager();
 
 /**
- * Returns form states for the active mod. Includes both dirty elements tracked by the page editor, and other
- * components that are active on the page.
+ * Returns mod component form states for the active mod. Includes dirty mod component form states.
```

Comment:
```suggestion
 * Returns mod component form states for both the dirty mods as well as clean, activated mods
```

This function is used in analysis to standardize all the activated mod components into a common shape for page editor Analysis to run over everything. I suspect that, since it was already implemented for the active dirty mod component form state, this was the easiest way to run it over everything is just turn everything into a form state. It's only called in the `analysisManager` right now.

Reply:
But also it only returns the mod components for one mod at a time, if the active mod component is in a mod. If the active mod component is a standalone mod it just returns that component form state.

Reply:
I updated the comment; does that address your concerns?